### DISCUSSION TITLE
Blocage de l'apparition du small event league reward le samedi

### DISCUSSION THREAD

**Discussion by voltou91** [Crownicles/Crownicles#1973] [Nouvelles idées]
Première fois que je tombe dessus le samedi du coup je ne savais pas que ça faisais ça.
Mais je propose que le canBeExecuted check si on est samedi et si c'est le cas, regarde si tu peux avoir les league rewards. Car tomber sur ce message (et se faire bouffer un mini-event) alors qu'on sait qu'on a pas le league reward, bon...
![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/62652451/115e6e77-ec8d-4d69-ba94-b7f6f106c0b4)

**Comment by Nargacaura** [comment]
T'es sûr d'être tombé dessus un samedi? Ça m'a l'air d'être tombé un dimanche, au vu du contenu du mini-event... (Sinon, est-ce que ça n'ressemble pas à une autre suggestion probablement noyée dans la liste — la #1870?)

**Comment by voltou91** [comment]
Oula je croyait on était samedi aujourd'hui XD
Mais du coup oui aujourd'hui

**Reply by QQtin3** [reply]
Bro is drunk

**Comment by BastLast** [comment]
Faudrait rajouter ube catégorie duplicata dans 'es suggestions enft parce quevya tellement de gens qui renvoient la mm chose mdr
